File: Shuffle-the-Array.cpp
1class Solution(object):
2    def shuffle(self, nums, n):
3        """
4        :type nums: List[int]
5        :type n: int
6        :rtype: List[int]
7        """
8        nums1 = []
9        for i in range(n):
10            nums1.append(nums[i])
11            nums1.append(nums[i+n])
12        return nums1
13        
14

File: 1580-shuffle-the-array/shuffle-the-array.py
class Solution(object):
    def shuffle(self, nums, n):
        """
        :type nums: List[int]
        :type n: int
        :rtype: List[int]
        """
        nums1 = []
        for i in range(n):
            nums1.append(nums[i])
            nums1.append(nums[i+n])
        return nums1
        
